projects/js-packages/publicize-components/src/components/social-post-modal/preview-section.tsx

```diff
@@ -79,7 +79,7 @@ export function PreviewSection() {
 		<div className={ styles[ 'preview-section' ] }>
 			<TabPanel tabs={ connections }>
 				{ ( tab: ( typeof connections )[ number ] ) => {
-					const isEnabled = canBeTurnedOn( tab ) && tab.enabled;
+					const isEnabled = Boolean( canBeTurnedOn( tab ) && tab.enabled );
```

Comment:
This is to fix this warning

<img width="960" alt="Screenshot 2025-03-11 at 3 59 05 PM" src="https://github.com/user-attachments/assets/91eb0c1d-213b-4907-9ce1-47c7d84d35ca" />